package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "grover",
-  "version": "1.1.10",
+  "version": "1.1.11",
```

Comment:
Generally best practice to leave version numbers as is and let the package maintainer worry about that 😄 

Not too much an issue with this PR, but for larger projects it would tend to lead to commit conflicts or potential issues with general release policy (ie should it be a patch release or a minor release etc).